### ISSUE TITLE
Does optimiser need to be parsed into the scheduler?

### ISSUE THREAD

**Issue by guydurant** [e-vissch/hydra_lightning_template#1]
`lr_scheduler = torch.optim.lr_scheduler.CosineAnnealingLR` in line 137 of cli.py does not have the optimiser as the param.

For me to get it to work, I added it as a param as well as other config params, is this neccessary?

`lr_scheduler = torch.optim.lr_scheduler.CosineAnnealingLR(
            optimizer,
            T_max=self.hparams.scheduler.T_max,
            eta_min=self.hparams.scheduler.eta_min,
        )`
